clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/ubam/TransferUbamV1QueryOutput.scala

```diff
@@ -14,36 +14,36 @@ case class TransferUbamV1QueryOutput(
   lane: Int,
   libraryName: String,
   location: Location,
-  analysisType: Option[String] = None,
-  baitIntervals: Option[String] = None,
-  baitSet: Option[String] = None,
-  dataType: Option[String] = None,
+  analysisType: Option[Symbol] = None,
+  baitIntervals: Option[Symbol] = None,
```

Comment:
This is going to point to a file, so `URI` might be more appropriate... but also it'll be a reference file that Clio should never touch, so maybe `Symbol` is OK to keep it pretty abstract. What does reviewer 2 think?

Reply:
(We can also decide to swap it later without rebuilding search, since both `Symbol` and `URI` map to a keyword field)

Reply:
I would keep `Symbol` since it's opaque data to clio.